python/lsst/ip/diffim/subtractImages.py

```diff
@@ -273,12 +273,25 @@ def run(self, template, science, sources, finalizedPsfApCorrCatalog=None):
                                             finalizedPsfApCorrCatalog=finalizedPsfApCorrCatalog)
         checkTemplateIsSufficient(template, self.log,
                                   requiredTemplateFraction=self.config.requiredTemplateFraction)
-        sciencePsfSize = getPsfFwhm(science.psf)
-        templatePsfSize = getPsfFwhm(template.psf)
+
+        fwhmExposureBuffer = self.config.makeKernel.kernel.active.fwhmExposureBuffer
+        fwhmExposureGrid = self.config.makeKernel.kernel.active.fwhmExposureGrid
```

Comment:
Just `self.config.makeKernel.fwhmExposureBuffer` if you move the parameter definition.